src/mosquito/base.cr

```diff
@@ -7,6 +7,9 @@ module Mosquito
     class_getter scheduled_tasks = [] of PeriodicTask
     class_getter timetable = [] of PeriodicTask
 
+    # Minimum time in seconds to wait between checking for jobs in redis.
+    class_property idle_wait : Float64 = 0.1
```

Comment:
Given that this will only ever be used by the runner, why not just define it on the runner itself rather than the base class?